### PR TITLE
Fastapi

### LINKED ISSUE #37: user mail validieren !!

### PR DESCRIPTION
**Pull request by scientes**
change from flask to fastapi
basic implement #37

### CODE REVIEW

**Review by Alienora — APPROVED** [review]

File: control_service/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-Flask-Caching
-psycopg2-binary
+fastapi
+databases
+aiosqlite
+passlib[bcrypt]
+python-multipart
+pyjwt
+aiodns
+postgresql
```
Comment (vilaureu) [inline control_service/requirements.txt]: There is no package `postgresql` (according to *pip* on `docker-compse build`). Did you mean `py-postgresql`?

File: control_service/control_service/routers/users.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+@router.post('/Register')
+async def register(data: RegisterUser):
+
+    response = {}
+    query = sql_userdata.select().where(sql_userdata.c.UserName == data.UserName)
+    query2 = sql_userdata.select().where(sql_userdata.c.Email == data.Email)
+    result = await database.fetch_one(query)
+    result2 = await database.fetch_one(query2)
```
Comment (vilaureu) [inline control_service/control_service/routers/users.py:105]: You could use [create_task](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task) to wait for the database opperations concurrently.

**Review by vilaureu — CHANGES_REQUESTED** [review]
Apart from the requirements error this seems to work perfectly. Great work 👍.

File: control_service/requirements.txt (hunk 1 above)
Comment (scientes) [reply control_service/requirements.txt]: done

**Review by vilaureu — APPROVED** [review]

**Pull request merged**